vm/os-unix.cpp

```diff
@@ -491,6 +491,14 @@ void handle_ctrl_c() {
   sigaction_safe(SIGINT, &fep_sigaction, NULL);
 }
 
+void factor_vm::primitive_disable_ctrl_break() {
+  // TODO
+}
+
+void factor_vm::primitive_enable_ctrl_break() {
+  // TODO
+}
+
 void abort() {
```

Comment:
Think you should set `stop_on_ctrl_break = false;` and `stop_on_ctrl_break = true;` in these primitives.